lib/mobility-core/src/Kernel/External/Payment/Interface/Juspay.hs

```diff
@@ -501,3 +531,7 @@ mkOfferNotifyReq merchantId OfferNotifyReq {..} = do
       txn_status = transactionStatus,
       offers = offers <&> (\OfferNotifyOffer {offerId, status} -> Juspay.OfferNotifyOffer {offer_id = offerId, status})
     }
+
+---- Helper function ----
+posixTextToUTCTime :: Text -> UTCTime
```

Comment:
You can just use `posixSecondsToUTCTime` by passing `(read (T.unpack posixDate) :: Int)` from the caller instead of creating a new function